capstone.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Electrical Engineering Portal</title>
+    <link rel="stylesheet" href="capstone.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+
+
+
+<header>
+    <nav class="container">
+        <div class="upper-nav">
+                <a href="https://www.facebook.com/haftamudesta"><i class="fa-brands fa-4x fa-facebook"></i></a> 
+            <i class="fa-brands fa-twitter-square"></i>
+            <p>English</p>
+            <p>My page</p>
+            <p>Logout</p>
+        </div>
+        <div class="lower-nav">
+            <div class="logo">
+                <a href="#home"><img src="images/electric-power-engineers1.png" alt=""></a>
+                <img src="images/Enabled.png" class="hamburger" alt="hambergur menu">
```

Comment:
- [ ] Kindly make sure your logo linke]s to the home page 🙏 . Example ` a href="/capstone.html"`